binr/radiff2/radiff2.c

```diff
@@ -683,7 +683,7 @@ int main(int argc, char **argv) {
 	case MODE_DIFF:
 	case MODE_DIFF_STRS:
 	case MODE_DIFF_IMPORTS:
-		d = r_diff_new (0LL, 0LL);
+		d = r_diff_new ();
```

Comment:
oh fuck, that was because of the missing (void) in rdiff.h can you update that to avoid further issues like this?